### PR TITLE
Papa 8 Telemetered csvs

### PR DESCRIPTION
**Pull request by cdobs**
Creates ingestion csvs for Papa 8 telemetered data. These are placeholders since there were no open ocean gliders deployed to the array.

### CODE REVIEW

File: GP02HYPM/GP02HYPM_D00008_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+#mi.dataset.driver.WFP_ENG.wfp_sio.wfp_eng_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.we_wfp_1237122.dat,GP02HYPM-WFP03-00-WFPENG000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.flord_l_wfp.sio.flord_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.we_wfp_1237122.dat,GP02HYPM-WFP03-01-FLORDL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.dosta_ln.wfp_sio.dosta_ln_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.we_wfp_1237122.dat,GP02HYPM-WFP03-03-DOSTAL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.ctdpf_ckl.wfp_sio.ctdpf_ckl_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.wc_wfp_1237122.dat,GP02HYPM-WFP03-04-CTDPFL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.vel3d_l.wfp.sio.vel3d_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.wa_wfp_1237122.dat,GP02HYPM-WFP03-05-VEL3DL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_D00008_ingest.csv:13]: Looks good.

File: GP03FLMA/GP03FLMA_D00008_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#,/omc_data/whoi/OMC/GP03FLMA/D00008/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOH051,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMA/D00008/*status_1399801.dat,GP03FLMA-RIS01-00-SIOENG000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMA/D00008/*dosta_*.dat,GP03FLMA-RIS01-03-DOSTAD000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.phsen_abcdef.sio.phsen_abcdef_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMA/D00008/*phsen_*.dat,GP03FLMA-RIS01-04-PHSENF000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.flort_dj.sio.flort_dj_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMA/D00008/*flort_*.dat,GP03FLMA-RIS01-05-FLORTD000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
```
Comment (snwhite) [inline GP03FLMA/GP03FLMA_D00008_ingest.csv:20]: Looks good.

File: GP03FLMB/GP03FLMB_D00008_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#,/omc_data/whoi/OMC/GP03FLMB/D00008/*ctdmo_*.dat,GP03FLMB-RIM01-02-CTDMOH071,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMB/D00008/*status_1488301.dat,GP03FLMB-RIS01-00-SIOENG000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMB/D00008/*dosta_*.dat,GP03FLMB-RIS01-03-DOSTAD000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.phsen_abcdef.sio.phsen_abcdef_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMB/D00008/*phsen_*.dat,GP03FLMB-RIS01-04-PHSENF000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.flort_dj.sio.flort_dj_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMB/D00008/*flort_*.dat,GP03FLMB-RIS01-05-FLORTD000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
```
Comment (snwhite) [inline GP03FLMB/GP03FLMB_D00008_ingest.csv:20]: Looks good.

**Review by snwhite — APPROVED** [review]

File: GP02HYPM/GP02HYPM_D00008_ingest.csv (hunk 1 above)
Comment (rtravis-whoi) [reply GP02HYPM/GP02HYPM_D00008_ingest.csv:13]: Looks good - I think.  Not quite sure what the 'we, wc, wa' in filename mask are or where they come from, but they look the same as past deploy ingest sheets.

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**